Assets/SC_Inentory.cs

```diff
@@ -0,0 +1,76 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SC_Inentory : MonoBehaviour
+{
+    private bool _isFinish;
+
+    private bool salad;
+    private bool _bacon;
+    private bool CHEESE;
```

Comment:
These variables should have been named more clearly using camelCase starts with is (isOpen, isActive)